nexus/src/app/allow_list.rs

```diff
@@ -67,7 +74,8 @@ impl super::Nexus {
                     ));
                 }
             }
-            if !contains_remote {
+            if !contains_remote && !matches!(server_kind, ServerKind::Techport)
```

Comment:
Tiny nit - I've started to really dislike `matches!` because it's effectively a match with a `_ => false` arm, and I don't love catchall arms like that. What would you think about lifting this up above the loop into a full match; something like

```rust
let require_remote_addr = match server_kind {
    // can move the comment about why we treat this to here
    ServerKind::TechPort => false,
    ServerKind::External => true,
};
```

and now that I write that, I'm not sure what `ServerKind::Internal` should be? Probably `false` - if that's right, it's another ding against `matches!`.

Reply:
Done in 85d66c0ed